Index: src/main/java/poo/coders/main/Controller.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>package poo.coders.main;\r\n\r\npublic class Controller {\r\n\tpublic void testClick(){\r\n\t\tSystem.out.println(\"Clickeaste wacho.\");\r\n\t}\r\n}\r\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- src/main/java/poo/coders/main/Controller.java	(revision be0c6d969230aaf246d6eedcd888ebf406d65f7d)
+++ src/main/java/poo/coders/main/Controller.java	(date 1596678243033)
@@ -2,6 +2,6 @@
 
 public class Controller {
 	public void testClick(){
-		System.out.println("Clickeaste wacho.");
+		System.out.println("Chupala gil.");
 	}
 }
